lisa/sut_orchestrator/azure/common.py

```diff
@@ -1356,9 +1359,16 @@ def check_or_create_resource_group(
         if not az_shared_rg_exists:
             log.info(f"Creating Resource group: '{resource_group_name}'")
 
+            if managed_by == "":
+                managed_by_para = None
+            else:
+                log.info(f"Using managed_by resource group： '{managed_by}'")
+                managed_by_para = managed_by
+
             with global_credential_access_lock:
                 rm_client.resource_groups.create_or_update(
-                    resource_group_name, {"location": location}
+                    resource_group_name,
+                    {"location": location, "managed_by": managed_by_para},
```

Comment:
The dict can be assigned to a variable firstly, so you can base on if managed_by is set or not to update the dict.